### ISSUE TITLE
Preference Changes Require Screen Refresh (w Multiple Viewers)

### ISSUE THREAD

**Issue by cscottjames** [ableplayer/ableplayer#581]
When there are multiple viewers on the same page, changes to Caption preferences require a refresh to go into effect.

Steps to reproduce: create a page with multiple Ableplayer instances. Update the caption style (font size, color, etc.). View the captions.

There are also other issues with multiple viewers on the same page; for example, if you hover over buttons (i.e. play, pause, etc.) on a second viewer, the label will appear on the first viewer instead of the second.

Related tickets are https://github.com/ableplayer/ableplayer/issues/571 and https://github.com/ableplayer/ableplayer/issues/86. Resolving #571 doesn't seem to solve the problem, though.

**Comment by cscottjames** [comment]
You are correct. Switching .geteferences() for .loadCurrentPreferences() is the solution. No need to go in a different direction. At the time of the OP, the comments in 571 didn't mention .loadCurrentPreferences() and from the typo, it didn't seem like that was the intended method.

**Comment by joedolson** [comment]
Fixed in merging 583. Thank you!
